macOS/DuckDuckGo/Preferences/View/PreferencesAppearanceView.swift

```diff
@@ -121,6 +121,91 @@ extension Preferences {
         }
     }
 
+    // MARK: - Theme View
+    //
+    struct ThemeView: View {
+
+        // MARK: - Constants
+        private let outerSize: CGFloat = 42
+        private let outerCornerRadius: CGFloat = 8
+        private let innerSize = CGSize(width: 40, height: 31)
+        private let innerCornerRadius: CGFloat = 7
+        private let knobSize = CGSize(width: 28, height: 8)
+        private let knobRadius: CGFloat = 7
```

Comment:
I don't **normally** keep constants up like this, but **some** of them are required in more than one spot. The rest are there just for consistency, happy to change this if needed!